ios/build-help.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Uses the open-source `wget` utility to create an embedding-friendly offline mirror
+# equivalent of the online iOS help.
+
+HELP_ROOT=keyman/Keyman/resources/OfflineHelp.bundle/Contents/Resources
+
+if [ -z $VERSION ]; then
+  VERSION="12.0"  # Set manually here because alpha versions usually don't have 
```

Comment:
I think we want to avoid manually updating version strings in build scripts.

Can you normally use  `cat ../resources/VERSION.md`?
Then, if you have access to the environment `$TIER` and $TIER? is "alpha", decrement the version.

Reply:
I thought about it... decrementing a string's not the simplest thing, though.  I'll look into it.

Reply:
OK, got something working.

Reply:
Nice! I just remembered you also added a redirect on help.keyman.com for alpha versions to go to the stable version. So you've got it covered on both ends